### PR TITLE
Configure label selector

### PR DESCRIPTION
**Pull request by xh3b4sd**
We have to configure the CRD label selector for e2e tests here explicitly to unblock the e2e tests of the other provider operators.

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -53,7 +53,7 @@ required = [
   name = "github.com/giantswarm/e2e-harness"
 
 [[constraint]]
-  branch = "master"
+  branch = "optional-label-selector"
   name = "github.com/giantswarm/e2etemplates"
```
Comment (xh3b4sd) [inline Gopkg.toml]: See https://github.com/giantswarm/e2etemplates/pull/44.

File: integration/ipam/ipam.go [hunk 2]
```diff
@@ -37,7 +37,7 @@ func GenerateFlannelNetwork(ctx context.Context, clusterID string, crdStorage mi
 
 	cidrMask := net.CIDRMask(flannelCidrSize, 32)
 
-	cidr, err := ipamService.CreateSubnet(ctx, cidrMask, flannelNetworkAnnotation(clusterID))
+	cidr, err := ipamService.CreateSubnet(ctx, cidrMask, flannelNetworkAnnotation(clusterID), nil)
```
Comment (xh3b4sd) [inline integration/ipam/ipam.go]: The service got changed. Only adapting. I think the reservation could be reworked and improved but not doing this now.

File: integration/setup/common.go [hunk 3]
```diff
@@ -53,14 +55,35 @@ func common(config Config) error {
...
+		c := chartvalues.NodeOperatorConfig{
+			RegistryPullSecret: env.RegistryPullSecret(),
+		}
+
+		values, err := chartvalues.NewNodeOperator(c)
+		if err != nil {
+			return microerror.Mask(err)
+		}
+
+		err = config.Release.InstallOperator(ctx, "node-operator", release.NewStableVersion(), values, corev1alpha1.NewNodeConfigCRD())
```
Comment (xh3b4sd) [inline integration/setup/common.go]: I took the opportunity to install some resources the new way.

**Review by MarcelMue — APPROVED** [review]

File: integration/setup/common.go (hunk 3 above)
Comment (kopiczko) [inline integration/setup/common.go]:
```suggestion
		err = config.Release.InstallOperator(ctx, "node-operator", release.NewStableChartInfo("node-operator-chart"), values, corev1alpha1.NewNodeConfigCRD())
```

**Review by kopiczko — APPROVED** [review]

**Pull request merged**